src/pages/public_registration/LogPanel.tsx

```diff
@@ -85,12 +86,14 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
             }),
             filesInfo: getTicketFilesInfo(publishingTicket, registrationFiles),
             type: 'PublishingRequest',
+            finalizedBy: publishingTicket.finalizedBy,
```

Comment:
Vet du om `finalizedBy` får noen verdi om institusjonen tillater registratorer å publisere filer selv? Hvis den ikke settes burde vi kanskje ha en fallback til `publishingTicket.owner` her 🤔

Reply:
`finalizedBy` vil være den brukeren som trykker på knappen godkjenn/avvis. Så om registrator har mulighet til dette, så vil denne personen dukke opp som `finalizedBy`. Skal dobbeltsjekke om det jeg sier er sant da :)

Reply:
I tilfellene hvor registrator selv har lov til å publisere vil de automatisk bli satt til publisert når man legger til ei ny fil på posten. Mener det da blir opprettet en ticket som automatisk blir satt til "Completed". Usikker, men tipper den da vil være uten `finalizedBy`

Reply:
Ja, sant, nå som du sier det så oppdaget jeg den buggen(?) i går, og lagde en backend-task i morgest: [NP-46825](https://unit.atlassian.net/browse/NP-46825). Det blir riktig når det er publicationOwner selv som er kurator og laster opp fil (veldig corner case). Men om det er en annen kurator som laster opp en fil, så vil ticket bli automatisk approved, med resourceOwner som feilaktig finalizedBy. Så den casen blir fikset i den PR'en 👍

Reply:
En antagelser her, men tror det kan være en vanlig arbeidsflyt for mindre institusjoner at man har en kurator som også legger inn mange av resultatene selv. Den casen jeg tenkte på var egentlig casen hvor en _ikke-kurator_, altså helt vanlig registrator, har lov til å publisere filer selv. Vet ikke hvordan backend håndterer det. (Dette gjelder altså worflow 1, hvor registrator kan publisere både metadata og filer)

Reply:
God presisering. Har oppdatert [NP-46825](https://unit.atlassian.net/browse/NP-46825) til å reflektere dette 👍